console/src/main/scripts/plugins/metrics/html/app-details/detail-platform.html

```diff
@@ -64,13 +64,13 @@ <h3 class="pull-left">Metrics</h3>
         <h2>Memory Usage</h2>
         <div class="hk-graph-container hk-graph-metrics" ng-if="os.resolvedMemoryData">
           <!-- HINT: colors for the chart can be changed in the hawkular-charts.css -->
-          <hawkular-chart
+          <hk-metric-chart
             data="os.chartMemoryUsageData"
             chart-type="area"
             hide-high-low-values="false"
             y-axis-units="Available (MB)"
             chart-height="250">
```

Comment:
`chart-height` is no longer used because it is fully responsive so it fills whatever dimensions the parent container provides.